COMPOUNDING_RATES.md

```diff
@@ -0,0 +1,149 @@
+# Terms
+
+Swivel v3 integrates with multiple lending protocols, including:
+
+- Compound
+- Rari
+- Aave
+- Euler
+- Yearn
+- ERC-4626 based lending protocols
+
+Each of these lending protocols have slightly different mechanics/implementations. Swivel tries to abstract these
+differences away for users and provide a single entry point to fixed and amplified yield lending on either of these
+protocols.
+
+## Compounding Token
+
+When supplying underlying to a lending protocol, one typically receives a token in exchange which can be redeemed at a 
+later time for the amount of underlying lent plus any interest generated over the lending period.
+
+Different lending protocols issue different tokens, e.g.
+
+- Compound: cToken (https://compound.finance/docs/ctokens)
+- Rari: fToken (https://docs.rari.capital/fuse/#ftoken-s)
+- Aave: aToken (https://docs.aave.com/developers/tokens/atoken)
+- Euler: eToken (https://docs.euler.finance/developers/contract-reference#ieuleretoken)
+- Yearn: yVault Token (https://docs.yearn.finance/getting-started/products/yvaults/vault-tokens)
+- ERC-4626: https://eips.ethereum.org/EIPS/eip-4626
+
+Swivel coins the term "Compounding Token" to refer to these tokens in general, emphasizing the notion that these tokens
+accrue (*compound*) interest over time. The mechanics behind each of the different compounding tokens might be different,
+but a common denominator is the idea, that compounding tokens can be exchanged at a later time for the amount of
+underlying lent plus a corresponding amount of accrued interest.
+
+One can understand a compounding token as a **share** of the total amount of underlying in a lending protocol's 
+pool/market.
+
+## exchangeRate
```

Comment:
Honestly I had to just comment after reading this a few time how well its described. Wish I had criticism but this is concise and accurate.